bgpd/bgp_zebra.c

```diff
@@ -2866,7 +2866,7 @@ static int bgp_zebra_process_local_macip(ZAPI_CALLBACK_ARGS)
 		zlog_debug(
 			"%u:Recv MACIP %s f 0x%x MAC %pEA IP %pI4 VNI %u seq %u state %d ESI %s",
 			vrf_id, (cmd == ZEBRA_MACIP_ADD) ? "Add" : "Del", flags,
-			&mac, &ip, vni, seqnum, state,
+			&mac, &ip.ipaddr_v4, vni, seqnum, state,
```

Comment:
looks like `ip` could be either v4 or v6, so maybe this should be `%pIA` ?

Reply:
I don't know, it is possible. Is there a format option for `ipaddr`? Where did you see `%pIA`?

Since the original code was already using `%pI4` I assumed it is IPv4.

Reply:
I'll bet the change to PI4 came after the new noisy xref report in the builds, so it's recent. pIA is in lib/ipaddr.h...

Reply:
I updated the PR to use `%pIA`, thanks!

Reply:
Now we'll see how CI does...

Reply:
> Is there a format option for `ipaddr`?

`git grep 'pragma FRR' '*.h'` :smiley: